python-squid/src/test/java/org/sonar/python/parser/python_v3/PythonV3Test.java

```diff
@@ -45,4 +45,14 @@ public void function_declaration(){
     assertThat(p).notMatches("lambda x:Type: x");
   }
 
+  @Test
+  public void function_star_args(){
+    setRootRule(PythonGrammar.TYPEDARGSLIST);
+    assertThat(p).matches("*, foo=bar");
+    assertThat(p).matches("*args, boo");
+    assertThat(p).notMatches("*, boo");
+    assertThat(p).notMatches("*");
+    assertThat(p).notMatches("**");
```

Comment:
The last one raises a SyntaxError in 3.4 and 3.5

    >>> def c(p = 1, p2, *, ** kargs):
    ...  pass
    ... 
      File "<stdin>", line 1
    SyntaxError: non-default argument follows default argument